### DISCUSSION TITLE
Sins version v0.107.42 i see %eth0 after a fe80 address

### DISCUSSION THREAD

**Discussion by Glassertje** [AdguardTeam/AdGuardHome#6533] [General]
Sins version v0.107.42 after every fe80 address i see %eth0. If i click on it my log is emty. I first must delete %eth0 before i see the log of that client. Just updated to v0.107.43. I have this issue on 2 Raspberry Pi's with AGH.

Does other having the same issue? no idea how to make a github bug report, so that is why i ask here.

![vb](https://github.com/AdguardTeam/AdGuardHome/assets/114856327/f0291cd1-e5ae-4236-8a60-4c3a39702fc8)

**Comment by EugeneOne1** [comment]
@Glassertje, we've already received such reports (#6312, #6343, #6503), please consider following the first one.

**Comment by Glassertje** [comment]
> @Glassertje, we've already received such reports (#6312, #6343, #6503), please consider following the first one.

My apologies, I searched but didn't find anything because I didn't really know what to look for. Thank you.
